modules/serverless-sinkbinding-kn.adoc

```diff
@@ -70,6 +88,25 @@ spec:
                       fieldPath: metadata.namespace
 ----
 
++
+[NOTE]
```

Comment:
I might make this IMPORTANT instead of a note. I don't think you need the example anymore.